soak-tests/payments-activation.js

```diff
@@ -1,27 +1,160 @@
 import http from 'k6/http';
-import { check } from 'k6';
-import { sleep } from 'k6';
+import {check, sleep} from 'k6';
 
-export let options = {
+export const options = {
     stages: [
-        { duration: '1m', target: 70 }, // ramp up to 70 users
-        { duration: '28m', target: 70 }, // stay at 70 for ~30 min
+        { duration: '1m', target: 20 }, // ramp up to 20 users
+        { duration: '3m', target: 20 }, // stay at 20 for ~3 min
         { duration: '1m', target: 0 }, // scale down. (optional)
     ],
 };
 
+function setupV1() {
+    return "https://api.uat.platform.pagopa.it/api/checkout/payments/v1";
```

Comment:
these APIs will no longer be used (function -> proxy IO), I would say remove them from testing